### PR TITLE
Feat/add jira to sample app

### PR DESCRIPTION
**Pull request by mcalus3**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Screenshots attached (for UI changes)

Installation guide: https://github.com/RoadieHQ/backstage-plugin-jira

How it looks like:
https://www.loom.com/share/f7b76b8e3bdf46b68aadb0fa61abf6ff

### CODE REVIEW

File: .changeset/rude-eyes-crash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'example-app': patch
```
Comment (benjdlambert) [inline .changeset/rude-eyes-crash.md:2]: Don't need a changeset for this. It's never published to `npm` 👍

**Comment by benjdlambert** [comment]
Another thought is that i'm not sure that it's valuable in the `example-app` in the repo. Maybe we could add it with some data to the backstate demo but not sure it brings any value in it's current state?

**Comment by dtuite** [comment]
Hey @benjdlambert. The idea of the PR is to enable a discussion about whether or not it makes sense to have in the sample app. Totally agree that it might make more sense in the demo if we back it off to a JIRA project with some demo data.

**Comment by benjdlambert** [comment]
Yeah I think with this proxy config too if it's added to the sample app people will have to have a JIRA_URL to start the app which is a little annoying. 

Thinking this would be best placed on the demo site with a fake jira project or something to show the features if anything.

**Comment by dtuite** [comment]
@benjdlambert 👍  Probably best to close this.

**Pull request closed without merging**